### PR TITLE
Adds centos to Makefile for tests

### PR DESCRIPTION
**Pull request by JoshVanL**
```release-note
NONE
```

/assign @simonswine

### CODE REVIEW

File: puppet/modules/vault_server/Makefile [hunk 1]
```diff
@@ -6,11 +6,10 @@ verify: bundle_install
 bundle_install:
 	bundle install --path $(BUNDLE_DIR)
 
-acceptance: bundle_install
-	bundle exec rake beaker:default
+acceptance: accetance-centos
 
-# provide backwards compatiblity
-acceptance_three_node: acceptance-three-node
```
Comment (simonswine) [inline puppet/modules/vault_server/Makefile]: I guess we never ran those?

/assign @JoshVanL 
/unassign

Comment (simonswine) [reply puppet/modules/vault_server/Makefile]: Also doesn't make too much sense, as it's active/passive/passive

**Comment by JoshVanL** [comment]
Unsure why this is failing, it's passing locally. Will investigate further

File: puppet/modules/vault_server/templates/vault.service.erb [hunk 2]
```diff
@@ -5,6 +5,7 @@ After=consul.service vault-assets.service
 
 [Service]
 Environment=AWS_REGION=<%= @region %>
+ExecStartPre=/bin/sh -c "while true; do STATUS=$(curl -s -o /dev/null -k -w '%{http_code}' --request GET http://localhost:8500/v1/health/node/<%= @hostname %>); if [ $STATUS -eq 200 ]; then break; else sleep 5; fi; done"
```
Comment (simonswine) [inline puppet/modules/vault_server/templates/vault.service.erb]:
```
// check for acl readiness
 req, err = http.NewRequest("GET", fmt.Sprintf("%s/v1/acl/token/self", c.HTTPHostAddr), nil)
 if err != nil {
        return fmt.Errorf("error building acl readiness request: %v", err)
}
```

Node healthy does not mean the ACL system is ready. Can you please test it like that?

Also the whole hostname thing is quite ugly

**Pull request merged**